frontend/src/components/priceStaticBar/PriceStaticBar.tsx

```diff
@@ -35,8 +47,44 @@ export default function PriceStaticBar({ ...props }: IPriceStaticBar) {
   if (pathname === PATH.home || pathname === PATH.trim) {
     return <></>;
   }
+
+  const handleMouseDown = (event: React.MouseEvent) => {
+    dragRef.current = true;
+    const element = barRef.current!.getBoundingClientRect();
+    setStartX(event.clientX - element.left);
+    setStartY(event.clientY - element.top);
+  };
```

Comment:
전체적으로 구조 분해를 이용하면 좋을거같아요
ex) 
```suggestion
  const handleMouseDown = ({clientX, clientY}: React.MouseEvent) => {
    dragRef.current = true;
    const element = barRef.current!.getBoundingClientRect();
    setStartX(clientX - element.left);
    setStartY(clientY - element.top);
  };
```